staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -307,6 +313,7 @@ func (s *APIAggregator) RemoveAPIService(apiServiceName string) {
 	}
 	s.GenericAPIServer.Handler.NonGoRestfulMux.Unregister(proxyPath)
 	s.GenericAPIServer.Handler.NonGoRestfulMux.Unregister(proxyPath + "/")
+	s.openAPIAggregator.removeApiServiceSpec(apiServiceName)
```

Comment:
this is called from another thread/go-routine than the new controller. Both write to `s.openAPISpecs`, without locking 🙈